src/streaming-services/disneyplus/DisneyplusParser.ts

```diff
@@ -0,0 +1,111 @@
+import { Item } from '../../models/Item';
+import { registerScrobbleParser } from '../common/common';
+import { ScrobbleParser } from '../common/ScrobbleController';
+import { DisneyplusApi } from './DisneyplusApi';
+
+// TODO Cleanup
+
+export interface DisneyplusSession {
+	paused: boolean;
+	progress: number;
+}
+
+class _DisneyplusParser implements ScrobbleParser {
+	id: string;
+	title: string;
+	episodeTitle: string;
+	type: string;
+	season: number;
+	episode: number;
+	isCollection: boolean;
```

Comment:
Any particular reason for having `title`, `episodeTitle`, `type`, `season`, `episode` and `isCollection` here? It doesn't look like they're used anywhere?

Reply:
Disney Plus removes the elements from the DOM when hiding the title and controls.
If I remove them, it seems that the plugin is querying Trakt with unkown data.

I'm new to typescript, so not sure if this can be handled another way :)

Reply:
But you're not using these specific properties anywhere, so I don't see how it can affect anything. You only assign them [here](https://github.com/trakt-tools/universal-trakt-scrobbler/pull/65/files#diff-d1fd825e3b0b628976a42f935a29148422a5aaa028b171432183015a39da4ddaR91-R96). If you remove those lines, does it still query Trakt with unknown data?